### PR TITLE
Tendrils properly drop loot when hit by extreme explosions.

### LINKED ISSUE #44979: Explosions destroy tendrills without causing them to create a chasm/dropping loot

### PR DESCRIPTION
**Pull request by ghost**
:cl: floyd
fix: Tendrils properly drop loot when hit by extreme explosions.
/:cl:

also fixes that explosions apply damage weirdly

Fixes #44979

### CODE REVIEW

File: code/game/objects/structures/lavaland/necropolis_tendril.dm [hunk 1]
```diff
@@ -36,13 +36,9 @@ GLOBAL_LIST_INIT(tendrils, list())
 	gps = new /obj/item/gps/internal(src)
 	GLOB.tendrils += src
 
-/obj/structure/spawner/lavaland/deconstruct(disassembled)
+/obj/structure/spawner/lavaland/Destroy()
```
Comment (Dennok) [inline code/game/objects/structures/lavaland/necropolis_tendril.dm]: What if we need to destroy it with no chasm and loot?
I think here need make Destroy(force)
and if we have "force" dont spawn loot and chasm.
Or leave it as before, you make explosion dont hard qdel it.

**Comment by optimumtact** [comment]
\>Change it so deconstruct runs
\>then change the deconstruct to the destroy anyway

but why.tiff

**Comment by Qustinnus** [comment]
because it was originally in destroy and cyber boss moved it out for unknown reasons, there's still things that don't use apply damage to deconstruct the tendril that should drop loot @optimumtact

**Comment by optimumtact** [comment]
those cases are the bugs, there is a massive amount of overuse of qdel when destroying things that should not be tolerated, so I'd rahter fix them than move stuff into qdel.

qdel shouldn't be the equivalent of Death or Deconstruct on mobs/objs

**Comment by Dennok** [comment]
> because it was originally in destroy and cyber boss moved it out for unknown reasons, there's still things that don't use apply damage to deconstruct the tendril that should drop loot @optimumtact

I think no.
What if we want to destroy Z lvl?
you want to bunch runtimes and lavalend loot flying in cleared space?
If something qdel() thing instead of Death or Deconstruct, you need to fix using of qdel()

**Pull request merged**